### DISCUSSION TITLE
Data card or row request

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1809] [Proposals]
### Goal

I need to change the DIPA flows to adapt them to the new O2 guidelines. 

### Context

Currently we are using a boxed row with the "slot" for a link, the "preset" for checkbox plus "details" to display the amount ( view the visual proposal)
With the new guidelines of the font size for amounts and pricing, the title will be displayed with two lines. Furthermore, the new guidelines show these amounts close to the title.

### Impact

This proposal would create a data card or a row more fluid in order to adapt them to this new cases.

### Discussion Questions

Option A. Could we have a Row with two slots? I could show the amount and a link.
Option B. Could we have a Data card with a preset zone where the checkbox to select the card is displayed?

### Do you have a visual proposal?

https://www.figma.com/design/cTAAIPcY0GqS68hR7gBx0a/New-Brand---Debt-Payment-Methods---O2ES?node-id=6801-45030

**Comment by aweell** [comment]
> Option A. Could we have a Row with two slots? I could show the amount and a link.

Discussed Offline: The row has two slots, right and extra, the amount could be placed on extra slot to avoid having two text competing for the horizontal space. That way, the row can still keep the double interaction functionality.

> Option B. Could we have a Data card with a preset zone where the checkbox to select the card is displayed?

We already have an open issue to define selectable cards in the system: https://github.com/Telefonica/mistica-design/issues/1762. I link this discussion to the issue so we are aware of your use case when the definition starts.

**Comment by aweell** [comment]
I close this issue since the ideal solution is already covered in #1762
